test/unit/helpers/dom/Form.js

```diff
@@ -161,6 +161,10 @@ define(['okta', './Dom'], function (Okta, Dom) {
       return $('[data-type="save"]');
     },
 
+    backButton: function () {
```

Comment:
Isn't this already there in `AccountRecoveryForm`? We have `backToLoginButton` in AccountRecoveryForm